lib/Makefile

```diff
@@ -12,17 +12,27 @@ $(OBJECTS): %$(OBJEXT): %.c %.h
 clean-l:
 	rm -f $(OBJECTS)
 
+# Change baseDir in composer autogenerated files
+define fix_composer_paths
+	for file in autoload_psr4.php autoload_classmap.php autoload_files.php autoload_namespaces.php ; do \
+		sed -i "s#^\$$baseDir = .*#\$$baseDir = dirname('$(DESTDIR)$(domserver_webappdir)');#" $(1)/composer/$$file ; \
+	done
+	sed -i "s#__DIR__ . '/../../..'#'$(DESTDIR)$(domserver_webappdir)' . '/..'#" $(1)/composer/autoload_static.php
```

Comment:
I would make this a bit more robust by matching and replacing up to `/webapp/`. Then things won't break if the number of relative directories change or if some other file gets loaded from `/../../..`.